harness/testTypedArray.js

```diff
@@ -46,3 +46,365 @@ function testWithTypedArrayConstructors(f, selected) {
     }
   }
 }
+
+/**
+ * Helper for conversion operations on TypedArrays, the expected values
+ * properties are indexed in order to match the respective value for each
+ * TypedArray constructor
+ * @param  {Function} fn - the function to call for each constructor and value.
+ *                         will be called with the constructor, value and
+ *                         expected value.
+ * @return {[type]} - follows the return of testWithTypedArrayConstructors if
```

Comment:
`testWithTypedArrayConstructors` only returns `undefined`, so this documentation may be more distracting than helpful. It's probably safest to return nothing, so that future changes to `testWithTypedArrayConstructors` don't have unintended consequences here.